src/index.ts

```diff
@@ -917,5 +948,4 @@ promisifyAll(Bigtable, {
 module.exports = Bigtable;
```

Comment:
This is unrelated, but I don't like this :)  We can change this to proper TypeScript exports.